### DISCUSSION TITLE
Why can't I use the 'Global Network Map' in China?

### DISCUSSION THREAD

**Discussion by qinyihao** [GitSquared/edex-ui#1085] [Questions] [answered]
Hi there.
![image](https://user-images.githubusercontent.com/40647091/111037393-b85fa400-845e-11eb-8e27-ba4ff9483632.png)
As you can see in this image, the Global Network Map is unavailable. Can you enable it? Thanks.

By the way, my system is macOS 11.2.3, with Apple M1 chip, 16GB RAM.

**Comment by GitSquared** [comment, marked as answer]
Open the settings (ctrl+shift+s, or click on settings.json in file view) and change the `PingAddr` to something that you can access.

**Reply by qinyihao** [reply]
Thanks, I've solved that problem.
